src/types/types.js

```diff
@@ -265,6 +265,13 @@ export type EdgeMetadata = {
   notes?: string,
   amountFiat?: number,
   bizId?: number,
+  swap?: {
+    +pluginId: string,
+    +orderId: string,
+    +orderUri?: string,
+    +payoutAddress: string,
+    +payoutCurrencyCode: string
+  },
```

Comment:
Missing a lot of parameters and this also changed the naming proposed in the design doc. These were the following parameters for swap:

orderId
statusUrl
supportEmail
pluginId
pluginName
sourceCurrencyCode
sourceAmount (in units of exchange amount)
destinationWalletName
destinationCurrencyCode
destinationAmount (in units of exchange amount)
receiveAddress (this is user address used to receive exchange funds)
refundAddress
quoteType: 'fixed' | 'variable'

Reply:
We have a new design document that this follows.